docs/manual/scala/guide/production/ProductionOverview.md

```diff
@@ -0,0 +1,46 @@
+# Production
+
+Lagom doesn't prescribe any particular production environment, however out of the box support is provided for [Lightbend ConductR](https://www.lightbend.com/products/conductr).
+
+## Deploying Lagom services to ConductR
+
+Lightbend ConductR is a perfect match for Lagom, as it provides the following features:
+
+* a means to manage configuration distinctly from your packaged artifact;
+* consolidated logging across many nodes;
+* a supervisory system whereby if your service(s) terminate unexpectedly then they are automatically restarted;
+* the ability to scale up and down with ease and with speed;
+* handling of network failures, in particular those that can lead to a split brain scenario;
+* automated seed node discovery when requiring more than one instance of your service so that they may share a cluster;
+* the ability to perform rolling updates of your services;
+* support for your services being monitored across a cluster; and
+* the ability to test your services locally prior to them being deployed.
+
+To deploy your Lagom services using ConductR, see [[ConductR]].
+
+## Deploying to other platforms
+
+Lagom sbt support leverages the [sbt-native-packager](http://www.scala-sbt.org/sbt-native-packager/) to produce archives of various types. By default zip archives can be produced, but you can also produce tar.gz, MSI, debian, RPM, Docker and more.
+
+If using Maven, there are many plugins for Maven to produce artifacts for various platforms.
+
```

Comment:
I'm wary of investing too much in new lightbend-markdown features given paradox is what we're standardising on.

Note that from experience, the copy/paste maintenance burden is not bad.  Porting the docs, where it's the most work, is only done once.  After that, the maintenance tends to be really quite simple.  And another thing, because the barrier to contribution to our docs is so low, we tend to find the community is very fast at correcting errors like this.  So it's never been an issue that we've every really needed to solve.